crates/iota-json-rpc-tests/tests/transaction_builder_api.rs

```diff
@@ -298,49 +298,53 @@ async fn test_pay_all_iota() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
-#[sim_test]
-async fn test_publish() -> Result<(), anyhow::Error> {
-    let cluster = TestClusterBuilder::new().build().await;
-    let http_client = cluster.rpc_client();
-    let address = cluster.get_address_0();
-
-    let objects = http_client
-        .get_owned_objects(
-            address,
-            Some(IotaObjectResponseQuery::new_with_options(
-                IotaObjectDataOptions::new()
-                    .with_type()
-                    .with_owner()
-                    .with_previous_transaction(),
-            )),
-            None,
-            None,
-        )
-        .await?;
-    let gas = objects.data.first().unwrap().object().unwrap();
-
-    let compiled_package =
-        BuildConfig::new_for_testing().build(Path::new("../../examples/move/basics"))?;
-    let compiled_modules_bytes =
-        compiled_package.get_package_base64(/* with_unpublished_deps */ false);
-    let dependencies = compiled_package.get_dependency_storage_package_ids();
-
-    let transaction_bytes: TransactionBlockBytes = http_client
-        .publish(
-            address,
-            compiled_modules_bytes,
-            dependencies,
-            Some(gas.object_id),
-            100_000_000.into(),
-        )
-        .await?;
+mod example {
```

Comment:
Same reasoning as above
```suggestion
mod move {
```